packages/commonwealth/server/routes/updateNewProfile.ts

```diff
@@ -70,6 +80,48 @@ const updateNewProfile = async (
     },
   );
 
+  if (tag_ids) {
```

Comment:
This is almost the exact same logic as the updateCommunitityCategory handler, so it should be refactored into a separate helper function

Reply:
Done [here](https://github.com/hicommonwealth/commonwealth/pull/7824/commits/b5a4f0c5805f8cdb89f350c6d1d7670e04789712)